docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 Welcome to PySPI's documentation!
```

Comment:
Might need to change this (and a few other references) to lowercase `pyspi` if we're going to do below. I'll have a think..

Reply:
I changed the references in all of the source files to `pyspi` where they still said `PySPI`.